PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetHUtils.h

```diff
@@ -126,12 +129,30 @@ class AliAnalysisTaskEmcalJetHUtils {
   static double LHC15oEtaEfficiency(const double trackEta, const double params[13]);
   static double LHC15oEtaEfficiencyImpl(const double trackEta, const double params[13], const int index);
 
+  // LHC18qr efficiency parameters
+  // pt parameters
+  static const double LHC18qrParam_0_10_pt[11];                    //!<! 0-10% pt parameters
+  static const double LHC18qrParam_10_30_pt[11];                   //!<! 10-30% pt parameters
+  static const double LHC18qrParam_30_50_pt[11];                   //!<! 30-50% pt parameters
+  static const double LHC18qrParam_50_90_pt[11];                   //!<! 50-90% pt parameters
+  // Eta parameters
+  static const double LHC18qrParam_0_10_eta[13];                   //!<! 0-10% eta parameters
+  static const double LHC18qrParam_10_30_eta[13];                  //!<! 10-30% eta parameters
+  static const double LHC18qrParam_30_50_eta[13];                  //!<! 30-50% eta parameters
+  static const double LHC18qrParam_50_90_eta[13];                  //!<! 50-90% eta parameters
+  // Helper functions for determining the LHC15o tracking efficiency
+  static double LHC18qrPtEfficiency(const double trackPt, const double params[11]);
+  static double LHC18qrLowPtEfficiencyImpl(const double trackPt, const double params[11], const int index);
+  static double LHC18qrHighPtEfficiencyImpl(const double trackPt, const double params[11], const int index);
+  static double LHC18qrEtaEfficiency(const double trackEta, const double params[13]);
+  static double LHC18qrEtaEfficiencyImpl(const double trackEta, const double params[13], const int index);
 };
 
 /**
  * All functions in the FlowVectorCorrections static class are copied diretly from the AddTask. I wish I didn't have
  * to do this, but I need it in compiled code, so I have no choice.
  */
+/*
```

Comment:
I guess you don't need this functionality, which is fine. If it's not preventing the code from compiling, you should remove the comment

Reply:
Removed everything below line 150